packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -3,42 +3,39 @@ import filter from 'lodash/filter';
 import get from 'lodash/get';
 import map from 'lodash/map';
 import sortBy from 'lodash/sortBy';
+import animateScrollTo from 'animated-scroll-to';
 import { API_GUIDES } from '../../../project.constants';
 
-import {
-  NAME_PATTERN,
-  MAX_NAME_LENGTH,
-  MIN_NAME_LENGTH,
-} from './add.constants';
 import { ENGINES_STATUS } from '../../../../../components/project/storages/databases/engines.constants';
+import { ENGINE_LOGOS } from '../databases.constants';
+import { ORDER_KEYS, nameGenerator } from './add.constants';
 
 export default class {
   /* @ngInject */
   constructor(
-    $anchorScroll,
     $translate,
-    $q,
     CucCloudMessage,
     DatabaseService,
-    Poller,
+    ovhManagerRegionService,
+    $scope,
```

Comment:
probably mistake, i don't see any usage .
To remove ?

Reply:
You're right, we do not use `$scope` anymore here, I removed it : https://github.com/ovh/manager/pull/6905/commits/15ac9b6b17cc909010729c53281b391ff78b7e59